### ISSUE TITLE
the mAP of SSD300 is lower than mAP in the paper ??

### ISSUE THREAD

**Issue by qwenqw** [weiliu89/caffe#695]
Hi, I just trained the pretrained VGG model on VOC 07+12. However,  after 120K iterations, the mAP is only 68%. In the paper, the SSD300 achieved 74.3%, even 77.2%  by more data augmentation for small object accuracy. And, I just used the file "examples/ssd/ssd_pascal.py" without any change. I don't know where is wrong ?
Anyone can help me? Thank you!

Please use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) for usage, installation, or modeling questions, or other requests for help.
_Do not post such requests to Issues._ Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md) before submitting this issue.

### Steps to reproduce

If you are having difficulty building Caffe or training a model, please ask the caffe-users mailing list. If you are reporting a build error that seems to be due to a bug in Caffe, please attach your build configuration (either Makefile.config or CMakeCache.txt) and the output of the make (or cmake) command.

### Your system configuration
Operating system:
Compiler:
CUDA version (if applicable):
CUDNN version (if applicable):
BLAS:
Python or MATLAB version (for pycaffe and matcaffe respectively):

**Comment by youngwanLEE** [comment]
Did you use 4 GPUs?

**Comment by qwenqw** [comment]
@youngwanLEE Thanks for your reply. I used only one GTX1080Ti. So  what should I change for the mAP in paper? Please give me more instruction! Thank you!

**Comment by youngwanLEE** [comment]
@qwenqw, Could you let me know batch size and decay step?
When you use only one GPU, you might lower the number of batch size per device then have to increase decay step.

**Comment by xuyunlu1030** [comment]
I just used the provided data and the file "examples/ssd/ssd_pascal.py" without any change. My loss is always around 100 even after training 30,000 iterations.  I don't know where is wrong, can you help me?

**Comment by qwenqw** [comment]
@youngwanLEE , I trained it again yesterday. And I set the parameters as follows: batch_size: 8, base_lr: 0.0001, stepvalue: [320000, 400000, 480000], gamma: 0.1, momentum: 0.9. After 480k iterations, the mAP achieved 74%. Can you help me adjust the parameters for higher accuracy? Thank you very much!

**Comment by qwenqw** [comment]
@youngwanLEE By the way, I noticed that you have implemented soft-NMS with SSD. And may I make so bold as to ask how about the results?

**Comment by qwenqw** [comment]
@xuyunlu1030 Hi, it's a little weird. When I begin training, the loss is around 20. Maybe you can try to check the inputdata or the pretrained model.

**Comment by xuyunlu1030** [comment]
@qwenqw I checked the input data. It seems no problem. I have no idea about that.
When I begin training, the loss is around 15. It decreased at the very beginning and than became larger and larger, range from 20 to 200. Data and pretrained model are all from the links. Are you training using the latest version of data, model and codes?  Or did you make some changes to the .py file?

**Comment by 408550969** [comment]
do you use the pretrained caffemodel? I have the same result when I didn't use the VGG_ILSVRC_16_layers_fc_reduced.caffemodel

**Comment by xuyunlu1030** [comment]
@qwenqw @408550969 Thank, it runs well when I reset my environment. I used the pretrained model and only changed the batch_size, maybe too small batch_size have influenced the training process. I'm not sure about the reason, though all is well now.

**Comment by lcwyylcwyy** [comment]
If you train your network from scratch, the performance will lower than the paper. And some resent paper propose the DSOD network to solute this issue.

**Comment by qwenqw** [comment]
@xuyunlu1030 Sorry to bother you again. Have you finished the training? And can you achieve the mAP of 77.2% in paper? I cannot achieve it. Can you give me some suggestions? Thank you very much.

**Comment by qwenqw** [comment]
Hello, all
I  try training the SSD300 model with one gpu (gtx1080Ti). 
And if I set batch_size=8, accum_batch_size=8, the mAP on VOC2007test is about 74% after 48w iterations.
If I set batch_size=8(or 32), accum_batch_size=32, the mAP is only 68%  after 12w iterations.
I am very confused. And I cannot solve it. Could you give me some solutions? Thanks a lot.
@weiliu89 @youngwanLEE

**Comment by sidgan** [comment]
@qwenqw Can you provide the link to the data augmentation script you are referring to.

**Comment by qwenqw** [comment]
@sidgan Thanks for your reply. I just used the default augmentation configurations in ssd_pascal.py. [https://github.com/weiliu89/caffe/blob/ssd/examples/ssd/ssd_pascal.py](url)

**Comment by RoseLii** [comment]
please help me .
I download VOCtrainval_06-Nov-2007，but when i running ./data/VOC0712/create_list.sh have error that
y@y-System-Product-Name:~/2T/wangxingli/caffe$ ./data/VOC0712/create_list.sh
Create list for VOC2007 trainval...
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
paste: /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
Create list for VOC2012 trainval...
cp: cannot stat '$/data/VOCdevkit/VOC2012/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
cp: cannot stat '$/data/VOCdevkit/VOC2012/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
paste: /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
Create list for VOC2007 test...
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/test.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_img.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_img.txt: No such file or directory
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/test.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_label.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_label.txt: No such file or directory
paste: /home/y/2T/wangxingli/caffe/data/VOC0712/test_img.txt: No such file or directory
I1025 21:36:51.506589 1277 get_image_size.cpp:61] A total of 0 images.
Why is this wrong?

**Comment by fanqinmin** [comment]
I have the same problem, have you solved it?
